src/yggdrasil/admin.go

```diff
@@ -536,6 +556,7 @@ func (a *admin) getData_getSelf() *admin_nodeInfo {
 	table := a.core.switchTable.table.Load().(lookupTable)
 	coords := table.self.getCoords()
 	self := admin_nodeInfo{
+		{"key", hex.EncodeToString(a.core.boxPub[:])},
```

Comment:
In other places in the admin socket, we use `box_pub_key` instead of `key` as the key name. I am not sure if we want to continue using that convention for clarity?